README.md

```diff
@@ -72,17 +76,17 @@ This is an experimental web client for Jellyfin based on Vue.js. We welcome all
    npm start
    ```
 
-   The client will be available at http://127.0.0.1:3000 by default.
+   The client will be available at <http://127.0.0.1:3000> by default.
 
-   ### Build for production
+### Build for production
 
-   When you're ready to deploy the client, you must build the client specifically production:
+When you're ready to deploy the client, you must build the client specifically production:
 
-   ```bash
-   npm run build
-   ```
+```bash
+npm run build
+```
 
-   Build output will be available under the `src/dist` folder.
+Build output will be available under the `src/dist` folder.
```

Comment:
Is the removal of the tabulation intended?

Reply:
yes that's the markdown linter that did it. And it seems to make sense to me

Reply:
I also did it in a specific commit :)

Reply:
@ThibaultNocchi How we can keep this layout then?

**master**
![image](https://user-images.githubusercontent.com/10274099/157510394-8fd77faf-2f36-41bc-a18b-1222bf0ccd52.png)

**Your branch**
![image](https://user-images.githubusercontent.com/10274099/157510458-87b7303d-6399-448a-9feb-325a70a852e8.png)

Imo it makes sense to keep the "Build for production" part inside "Getting Started" as it's the bare minimum an user needs to build their own client version. The rest of the sections cover more "advanced" use cases and are good as they're now, but imo, these two initial sections should be kept together.